src/app/features/config/global-config.model.ts

```diff
@@ -151,6 +152,7 @@ export type ScheduleConfig = Readonly<{
 export type ReminderConfig = Readonly<{
   isCountdownBannerEnabled: boolean;
   countdownDuration: number;
+  defaultTaskRemindOption: TaskReminderOptionId;
```

Comment:
We need to make this optional, since we are not migrating this explicitly. We also need to fallback to the AtStart value when this is not configured to avoid causing problems for some users.

Reply:
I had (mistakenly) assumed that there was some mechanism to automatically inherit the value defined in `DEFAULT_GLOBAL_CONFIG`. This should now be optional.